File: 2.1 insertion-sort/code.js
module.exports.sort = function sort(arr){
    let j = 1;
    for(; j < arr.length; j++){
        let key = arr[j];
        let i = j - 1;
        while(arr[i] > key && i >= 0){
            arr[i + 1] = arr[i];
            i--;
        }
        arr[i+1] = key;
    }
}

File: 2.1 insertion-sort/code.cc
void sort(int arr[], int size){
    int j = 1;
    for(; j < size; j++){
        int key = arr[j];
        int i = j - 1;
        while(arr[i] > key && key >= 0){
            arr[i + 1] = arr[i];
            i--;
        }
        arr[i+1] = key;
    }
}
